### PR TITLE
Finish monorepo merge: http-pouchdb

### PR DESCRIPTION
**Pull request by gr2m**
part of #197

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,7 +68,8 @@
     "serve-favicon": "~2.3.2",
     "tail": "^1.2.1",
     "uuid": "^3.0.1",
-    "wordwrap": "1.0.0"
+    "wordwrap": "1.0.0",
+    "xhr2": "^0.1.3"
```
Comment (gr2m) [inline package.json]: `xhr2` is a new dependency for http-pouchdb: https://github.com/pouchdb/http-pouchdb/blob/9b28231f2c40795c045956a6a4b0f3f974c661c9/package.json#L51

File: tests/http-pouchdb/test.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  });
+});
+
+describe('constructor', function () {
+  it('should create remote databases for normal db names', function () {
+    var users = new HTTPPouchDB('_users');
+
+    return users.info().then(function (info) {
+      info.should.have.property('db_name');
+    });
```
Comment (gr2m) [inline tests/http-pouchdb/test.js]: the `constructor` tests are different from the original: https://github.com/pouchdb/http-pouchdb/blob/9b28231f2c40795c045956a6a4b0f3f974c661c9/test/test.js#L25-L39

- For the unit tests, we start our own `pouchdb-server` so the `data_size` does not exist, hence I just check for `db_name`
-`new HTTPPouchDB('http://localhost:5984/_replicator')` is valid with the latest PouchDB, it just encodes all the slashes etc. If that is an error, it’s outside of the scope of `http-pouchdb`

**Comment by garrensmith** [comment]
@gr2m Awesome +1

**Comment by gr2m** [comment]
thanks for the review @garrensmith, I’ll move forward then :)

**Pull request merged**